src/core/namespace.c

```diff
@@ -64,6 +64,20 @@ typedef struct BindMount {
         bool ignore;
 } BindMount;
 
+typedef struct TargetMount {
+        const char *path;
+        MountMode mode;
+        bool ignore;
+} TargetMount;
+
+/* ProtectKernelTunables= option and the related filesystem APIs */
+static const TargetMount protect_kernel_tunables_table[] = {
+        { "/proc/sys",                  READONLY,       false, },
+        { "/proc/sysrq-trigger",        READONLY,       true, /* Not always compiled into kernel */ },
+        { "/sys",                       READONLY,       false, },
+        { "/sys/fs/cgroup",             READWRITE,      false, }, /* READONLY is set by ProtectControlGroups= option */
```

Comment:
Ouu probably I have the opposite of OCD which is probably the same thing ? updated with some medication but not sure if they are the right one! hehe